src/redux/features/observations/observationsSelectors.ts

```diff
@@ -0,0 +1,19 @@
+import { RootState } from 'src/redux/rootReducer';
+import { createSelector } from '@reduxjs/toolkit';
+
+export const selectObservationsResults = (state: RootState) => state.observationsResults?.observations;
+export const selectObservationsResultsError = (state: RootState) => state.observationsResults?.error;
+
+export const selectCompletedObservationsResults = createSelector(selectObservationsResults, (observationsResults) =>
+  observationsResults?.filter((observationResults) => !!observationResults.completedTime)
+);
+
+export const selectPlantingSiteObservationsResults = createSelector(
+  [selectCompletedObservationsResults, (state, plantingSiteId) => plantingSiteId],
```

Comment:
What does this list do?

Reply:
The observations page has a planting site selector. This list returns observations for the selected planting site - including 'All'.

Reply:
One thing about using selectors is that we use selectors to access data. That includes narrowing down of data.

Reply:
Got it, Thanks